{"status":"OK","subtests":[{"name":"Calling observe() without 'type' or 'entryTypes' throws a TypeError","status":"FAIL"},{"name":"Calling observe() with entryTypes and then type should throw an InvalidModificationError","status":"FAIL"},{"name":"Calling observe() with type and then entryTypes should throw an InvalidModificationError","status":"FAIL"},{"name":"Calling observe() with type and entryTypes should throw a TypeError","status":"FAIL"},{"name":"Passing in unknown values to type does throw an exception.","status":"FAIL"},{"name":"observe() with different type values stacks.","status":"FAIL"}]}